state-of-the-docs-2015.rst

```diff
@@ -0,0 +1,197 @@
+    bringing on more engineers, and engaging the community more, was the lack of
+    formal ticketing and open decision making.  Gregor, among many other
+    changes, laid out a formal triage process that has been a major influence in
+    keeping an organized and healthy support queue.
+
+    We've definitely found that the more support engagement we provide the
+    community, the more support comes out of the woodwork.  This is all part of
+    supporting a healthy community however, as we have learned.
+
+Stability
+    Read the Docs has "grown organically", which is to say it has grown fast,
+    unwieldy, and in many directions.  With more proper focus on code quality,
+    including clean up efforts, refactoring, and long due upgrades, Read the
+    Docs has shed considerable technical debt in the last couple months.  This
+    work has already paid off in improving stability and predictability of the
+    project.
+
+Gold
+    We decided to adopt a "gold" status feature model for users that want to
+    support us each month. This is an optional, recurring subscription to Read
+    the Docs.
+
+    We have plans to make gold subscriptions more functional, but for now, it is
+    just a way to show your support of Read the Docs.
+
+Ads
+    We ran a few tests with advertising on built documentation in the last few
+    months.  We discussed at length how we felt about ads and determined our own
+    problems with the standard implementation of ads is not the ads themselves,
+    but the breach of privacy they signify.
+
+    Because of that, we established that a requirement of us ever showing ads is
+    to not use third party tracking.  In fact, we look to remove all third party
+    tracking from built documentation as soon as we can.
+
+    To put ads into a clearer context, revenue from in-house ads fully cover our
+    costs each month, without asking the community for any further support. It
+    probably shouldn't be surprising that advertising revenue is an easier
+    income source than seeking donations. We certainly feel Read the Docs
+    provides more value than the ads it would serve though. Our last fundraiser
+    reflected the value the community puts on the site, and we hope the next
+    fundraiser can be as successful.
+
+    Should we ever use ads as a primary revenue source for the community site,
+    we have established some rules:
+
+    * We will provide our ads in-house, not through any service that establishes
+      third party tracking
+    * We will not sell any data about our users
+    * We will always prefer ad sponsorship from community members
+    * Users that support us through gold or sponsorship are opt out of ads
+
+    Feedback regarding ads has been positive, hopefully our intentions with ads
+    has been mostly apparent.
+
+The next year
+-------------
+
+So, let us now set some expectations for the next year forward:
+
+Monthly updates
```

Comment:
Whats the value of providing monthly updates? I feel like its overhead that doesn't buy us a lot -- seems like we should just be promoting new features/etc, rather than doing "updates".